routes/todo.go

```diff
@@ -0,0 +1,63 @@
+package routes
```

Comment:
You have a lot of repeated code every time you need to return a request, you should create a function that wraps this repeated code.

Reply:
Refactored b98bae7fcfe0da642a408313b957abf9639aac45